contracts/binding/src/lib.rs

```diff
@@ -2,31 +2,25 @@
 #[macro_use]
```

Comment:
is this change needed? I thought we forced this to be formatted in https://github.com/consensus-shipyard/ipc/pull/659